contracts/V1/DisputeTypes.sol

```diff
@@ -44,11 +68,22 @@ struct Proof {
 }
 
 enum ProofType {
-    FoldRechallenge,
+    // Block releated fraud proofs
     DoubleSign,
-    IncorrectData,
-    NewerState,
-    FoldPriorBlock,
+    EmptyBlock,
+    InvalidStateTransition,
+    OutOfGas,
+
+    // Timeout related fraud proofs
+    TimeoutThreshold,
+    TimeoutPriorInvalid,
+    TimeoutParticipantNoNext,
+
+    // Dispute fraud proofs
+    FoldRechallenge,
+    IncorrectData, // InvalidOutputState
+    NewerState, // NotLatestState
+    FoldPriorBlock, // challenge a false timeout proof
```

Comment:
I've renamed some fraud proofs in the docs so there's a naming convention (Block, Dispute,Timeout) - if a fraud proof is missing (some probably are, but will be added) just update the doc and here